Lombiq.Hosting.Tenants.FeaturesGuard/Handlers/FeaturesEventHandler.cs

```diff
@@ -14,168 +13,136 @@ namespace Lombiq.Hosting.Tenants.FeaturesGuard.Handlers;
 
 public sealed class FeaturesEventHandler : IFeatureEventHandler
 {
-    private readonly IShellFeaturesManager _shellFeaturesManager;
-    private readonly IOptions<ConditionallyEnabledFeaturesOptions> _conditionallyEnabledFeaturesOptions;
-    private readonly ShellSettings _shellSettings;
-    private readonly IShellDescriptorManager _shellDescriptorManager;
-
-    public FeaturesEventHandler(
-        IShellFeaturesManager shellFeaturesManager,
-        IOptions<ConditionallyEnabledFeaturesOptions> conditionallyEnabledFeaturesOptions,
-        ShellSettings shellSettings,
-        IConfiguration configuration,
-        IShellDescriptorManager shellDescriptorManager)
-    {
-        _shellFeaturesManager = shellFeaturesManager;
-        _conditionallyEnabledFeaturesOptions = conditionallyEnabledFeaturesOptions;
-        _shellSettings = shellSettings;
-        _shellDescriptorManager = shellDescriptorManager;
-    }
-
     Task IFeatureEventHandler.InstallingAsync(IFeatureInfo feature) => Task.CompletedTask;
 
     Task IFeatureEventHandler.InstalledAsync(IFeatureInfo feature) => Task.CompletedTask;
 
     Task IFeatureEventHandler.EnablingAsync(IFeatureInfo feature) => Task.CompletedTask;
 
-    Task IFeatureEventHandler.EnabledAsync(IFeatureInfo feature) => EnableConditionallyEnabledFeaturesAsync(feature);
+    Task IFeatureEventHandler.EnabledAsync(IFeatureInfo feature) => HandleConditionallyEnabledFeaturesAsync();
 
     Task IFeatureEventHandler.DisablingAsync(IFeatureInfo feature) => Task.CompletedTask;
```

Comment:
For all methods no need to use the interface name, just add the public modifier, for example

`Task IFeatureEventHandler.EnabledAsync()` can be replaced by `public Task EnabledAsync()`